src/library/tactic/simplify.cpp

```diff
@@ -478,7 +478,6 @@ simp_result simplify_core_fn::rewrite(expr const & e) {
 
     list<simp_lemma> const * srs = sr->find(e);
     if (!srs) {
-        lean_trace_d(name({"debug", "simplify", "try_rewrite"}), tout() << "no simp lemmas for: " << e << "\n";);
         return simp_result(e);
```

Comment:
Note: there are still many other gaps in the simplify-tracing scheme. Not every kind of congruence triggers a message, and no failures do when applying congruence. Making the tracing/debugging intuitive and robust is a priority for me, but I would like to discuss the overall simplify/converter/rewriter ecosystem with you next week before fine-tuning much further.